### PR TITLE
feat(account-lib): blockchain standard fixes

### PR DESCRIPTION
**Pull request by gibrangul95**
@fdmirza

### CODE REVIEW

File: modules/account-lib/src/coin/dot/transactionBuilder.ts [hunk 1]
```diff
@@ -58,15 +58,15 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
...
    * @returns {TransactionBuilder} This transaction builder.
    *
    * @see https://wiki.polkadot.network/docs/build-transaction-construction
    */
-  eraPeriod(eraPeriod: number | undefined): this {
-    if (eraPeriod !== undefined) {
-      this.validateValue(new BigNumber(eraPeriod));
-      this._eraPeriod = eraPeriod;
+  durationConfig({ maxDuration }: { maxDuration: number | undefined }): this {
+    if (maxDuration !== undefined) {
```
Comment (fdmirza) [inline modules/account-lib/src/coin/dot/transactionBuilder.ts:67]: maybe throw an error in else condition?

File: modules/account-lib/src/coin/dot/transactionBuilder.ts [hunk 2]
```diff
@@ -100,18 +101,19 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
...
    *
    * @param {number} blockNumber
    * @returns {TransactionBuilder} This transaction builder.
    *
    * @see https://wiki.polkadot.network/docs/build-transaction-construction
    */
-  blockNumber(blockNumber: number): this {
-    this.validateValue(new BigNumber(blockNumber));
-    this._blockNumber = blockNumber;
+  validity({ firstValid }: { firstValid: number }): this {
```
Comment (fdmirza) [inline modules/account-lib/src/coin/dot/transactionBuilder.ts:114]: Can we make an interface for this?

**Review by fdmirza — COMMENTED** [review]
Just a couple of comments, otherwise looks good.

File: modules/account-lib/src/coin/dot/transactionBuilder.ts (hunk 1 above)
Comment (gibrangul95) [reply modules/account-lib/src/coin/dot/transactionBuilder.ts:67]: we dont need to as this value is optional

**Pull request closed without merging**